Manifests/ManagedPreferencesApplications/com.secondsonconsulting.renew.plist

```diff
@@ -0,0 +1,468 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://github.com/SecondSonConsulting/Renew/releases</string>
+	<key>pfm_documentation_url</key>
+	<string>https://github.com/SecondSonConsulting/Renew/wiki</string>
+	<key>pfm_targets</key>
+	<array>
+		<string>system</string>
+	</array>
```

Comment:
This should be at the bottom, below `pfm_subkeys`

Reply:
The quickest way to apply @apizz's change requests would be to run the manifest through the following command on a Mac:
`plutil -convert xml1 com.secondsonconsulting.renew.plist`